lib/src/test/java/io/ably/lib/test/realtime/RealitimeReauthTest.java

```diff
@@ -0,0 +1,103 @@
+package io.ably.lib.test.realtime;
+
+import org.junit.Test;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.realtime.ConnectionState;
+import io.ably.lib.rest.AblyRest;
+import io.ably.lib.rest.Auth;
+import io.ably.lib.test.common.Helpers;
+import io.ably.lib.test.common.Setup;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.Capability;
+import io.ably.lib.types.ClientOptions;
+import io.ably.lib.types.ErrorInfo;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Created by VOstopolets on 8/26/16.
+ */
+public class RealitimeReauthTest {
+
+    /**
+     * use authorise({force: true}) to reauth with a token with a different set of capabilities
```

Comment:
It would be useful, if possible, to additionally add the spec item in the tests so that later other people can easily find out which spec(s) the test covers.